.test/config-tumor_only/config.yaml

```
@@ -9,14 +9,26 @@ ref:
   # Ensembl species name
   species: saccharomyces_cerevisiae
   # Ensembl release
-  release: 100
+  release: 111
   # Genome build
   build: R64-1-1
+  pangenome:
+      # if active, reads will be aligned to given pangenome instead of to the linear reference genome
+      # Important: this is only supported for homo_sapiens so far
+      activate: false
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)